arctic/arctic.py

```diff
@@ -447,6 +449,7 @@ def _auth(self, database):
             authenticate(database, auth.user, auth.password)
 
     def reset_auth(self):
+        logger.debug("reset_auth() %s" % self)
```

Comment:
we'd better move it in _auth() 
the reset_auth is not being used (much).

Reply:
since we no longer spam the auth with every call, it won't add much of a perf hit

Reply:
I'm already getting logs for calls into `_auth` as `authenticate` does a debug log on each call